docs/skopeo.1.md

```diff
@@ -12,26 +12,31 @@ It also allows you to copy container images between various registries, possibly
 ## IMAGE NAMES
 Most commands refer to container images, using a _transport_`:`_details_ format. The following formats are supported:
 
-  **atomic:**_namespace_**/**_stream_**:**_tag_
-  An image in the current project of the current default Atomic
-  Registry. The current project and Atomic Registry instance are by
-  default read from `$HOME/.kube/config`, which is set e.g. using
-  `(oc login)`.
+  **atomic:@**_hostname_**/**_namespace_**/**_stream_**:**_tag_
+  An image served by an OpenShift(Atomic) Registry server. The current OpenShift project and OpenShift Registry instance are by default read from `$HOME/.kube/config`, which is set e.g. using `(oc login)`.
+
+  **container-storage://**_docker-reference_
+  An image in local container/storage format.  Location and image store specified in /etc/containers/storage.conf
```

Comment:
This should be `containers-storage`, `containers/storage` (with an “s”)

Also, perhaps `s/format/image store/`? “format” to me implies one or more files which can be just copied around.